readme.md

```diff
@@ -1,8 +1,8 @@
 ## Tic Tac Toe
```

Comment:
Looks good! I think the only thing missing is a list of dependencies & setup/installation instructions (if there are any yet)

Reply:
Changes made. Should I put in an empty Setup/Installation section as a stub or just add it once it becomes relevant?

Reply:
I'm okay with adding once it becomes relevant; not sure what everyone else thinks